clients/java/src/main/java/io/zeebe/client/api/response/CreateWorkflowInstanceWithResultResponse.java

```diff
@@ -17,7 +17,7 @@
 
 import java.util.Map;
 
-public interface WorkflowInstanceResult {
+public interface CreateWorkflowInstanceWithResultResponse {
```

Comment:
This is a long name and feels kind of clunky :sweat_smile:

Reply:
will roll it back